Project.toml

```diff
@@ -5,27 +5,27 @@ version = "0.1.0-dev"
 
 [deps]
 DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```

Comment:
Why is documenter in the main Project.toml? It should only be needed in `docs/Project.toml`

Reply:
my bad.. fixed!